config/routes.rb

```diff
@@ -244,7 +244,7 @@
         end
 
         resources :lead_providers, only: %i[index show], path: "lead-providers"
-        resources :admins, only: %i[index]
+        resources :admins, except: :edit
```

Comment:
I think referring to 'admins' inside the admin section could possibly be a bit confusing. As these people will be DfE staff, maybe `/admin/staff`?

```suggestion
        resources :admins, except: :edit, path: `staff`
```